tests/test-basic.sh

```diff
@@ -21,6 +21,13 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
+# This test looks for specific localized strings. Prefer a UTF-8 locale
+if locale -a | grep '^C\.UTF-8$' >/dev/null; then
```

Comment:
For what it's worth, unconditionally setting either `LC_ALL=C` or `LC_ALL=C.UTF-8` seems to work fine on current Debian. Which would you prefer?